package.json

```diff
@@ -27,8 +27,22 @@
         "command": "angular.openLogFile",
         "title": "Open Angular Server log",
         "category": "Angular"
+      },
+      {
+        "command": "angular.getTemplateTcb",
+        "title": "View Template Typecheck Block",
+        "category": "Angular"
       }
     ],
+    "menus": {
+      "editor/context": [
+        {
+          "when": "resourceLangId == html || resourceLangId == ts",
+          "command": "angular.getTemplateTcb",
+          "group": "3_compare"
```

Comment:
One more thing we might want to consider: we can define our own group here if we wanted all Angular commands to appear together in the context menu.